### DISCUSSION TITLE
Some images are missing when a PDF is viewed in GitHub

### DISCUSSION THREAD

**Discussion by SimonORorke** [community/community#133329] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

One way to view my [application](https://github.com/SimonORorke/FalconProgrammer)'s PDF manual is via [a GitHub link](https://github.com/SimonORorke/FalconProgrammer/blob/main/Documentation/Falcon%20Programmer%20Manual.pdf).  When viewed via the link in a browser, some of the pictures the PDF contains are missing.  If the PDF is downloaded to a computer and opened there, no pictures are missing.

Looking for a pattern, I wondered if the missing images were all above some size.  But that's not it.  I would appreciate any suggestions as to how I might fix this.

**Comment by Sachinsingh2002** [comment]
hii @SimonORorke it could be due to several factors related to the PDF rendering in the browser.

Ensure that the images in your PDF are in a format and compression that is widely supported by web browsers, like JPEG and PNG are generally safe, but other formats might cause issues.

or you can use a PDF validator tool to check if the PDF adheres to the PDF/A standard, this can help identify any issues with the PDF structure that might cause rendering problems.

or if possible recreate the PDF from the original source document, sometimes the process of creating the PDF can introduce errors using a different PDF creation tool might resolve the issue.

**Comment by SimonORorke** [comment, marked as answer]
Thanks @Sachinsingh2002! Opting to adhere to the PDF/A standard when converting the source document to PDF has fixed the problem.  (The images are all PNGs.)

**Comment by Sachinsingh2002** [comment]
Hey @SimonORorke, that's great to hear! glad the PDF/A standard fixed the image issue, happy to help!
